src/views/tutorial-view/utils/get-collection-context.ts

```diff
@@ -1,27 +1,24 @@
 import {
-  ProductOption,
+  Collection as ClientCollection,
   TutorialFullCollectionCtx as ClientTutorialFullCollectionCtx,
 } from 'lib/learn-client/types'
 import { CollectionContext } from '..'
 import { formatCollectionCard } from '../components/featured-in-collections/helpers'
 
 export function getCollectionContext(
-  productSlug: ProductOption,
-  filename: string,
+  currentCollection: ClientCollection,
   collectionCtx: ClientTutorialFullCollectionCtx['collectionCtx']
 ): CollectionContext {
-  const collectionDbSlug = `${productSlug}/${filename}`
-  const current = collectionCtx.featuredIn.find(
-    ({ slug }) => slug === collectionDbSlug
-  )
-  const isDefault = current.id === collectionCtx.default.id
+  const isDefault = currentCollection.id === collectionCtx.default.id
   const featuredIn = collectionCtx.featuredIn
-    .filter((c) => c.id !== current.id)
+    .filter(
+      (c) => c.id !== currentCollection.id && !c.slug.includes('onboarding') // filter out onboarding content for CS team
```

Comment:
[See this ticket](https://app.asana.com/0/1126685961643737/1200827772802978) (and [this one](https://app.asana.com/0/search/1201991101510145/1200716871735242)) for the historical Learn context re: filtering the `onboarding` content.